cspp_runner/runner.py

```diff
@@ -569,50 +548,77 @@ def run(self, msg):
         LOG.info("Before call to spawn_cspp. Argument list = " +
                  str([keeper] + self.glist))
         LOG.info("Start time: %s", start_time.strftime('%Y-%m-%d %H:%M:%S'))
-        self.cspp_results.append(self.pool.apply_async(spawn_cspp,
-                                                       [keeper] + self.glist))
+        self.cspp_results.append(
+                self.pool.apply_async(
+                    spawn_cspp,
+                    [keeper] + self.glist,
+                    {"viirs_sdr_call": viirs_sdr_call,
+                     "viirs_sdr_options": viirs_sdr_options}))
         if self.fullswath:
             LOG.info("Full swath. Break granules loop")
             return False
 
         return True
 
 
-def npp_rolling_runner():
+def npp_rolling_runner(
+        thr_lut_files_age_days,
+        url_download_trial_frequency_hours,
+        lut_update_stampfile_prefix,
+        lut_dir,
+        url_jpss_remote_lut_dir,
+        mirror_jpss_luts,
+        url_jpss_remote_anc_dir,
+        anc_update_stampfile_prefix,
+        mirror_jpss_ancillary,
+        subscribe_topics,
+        site,
+        mode,
+        publish_topic,
+        level1_home,
+        viirs_sdr_call,
+        viirs_sdr_options,
+        granule_time_tolerance=10,
+        ncpus=1,
+        ):
```

Comment:
I started copying globals to a function argument, perhaps before I realised how many they would be.  But even if it was a class, it would just become the same number of arguments to the `__init__`...?

Reply:
I think it is fair to take small steps improving the code, so we can have a PR later to try solve the large amount of arguments! You could make an issue @gerritholl !